modules/lib/lib-schema/src/test/resources/test/UpdateDynamicContentSchemaHandlerTest.js

```diff
@@ -21,4 +21,403 @@ exports.updateInvalidContentSchemaType = function () {
+                              <label>Name</label>
+                              <help-text>Text input</help-text>
+                              <occurrences minimum="1" maximum="1"/>
+                            </input>
+                          </items>
+                        </option>
+                        <option name="option2_2">
+                          <label>Option 2_2</label>
+                          <default>true</default>
+                          <items>
+                            <input name="myCheckbox" type="Checkbox">
+                              <label>my-checkbox</label>
+                              <immutable>false</immutable>
+                              <indexed>false</indexed>
+                              <occurrences minimum="0" maximum="1"/>
+                            </input>
+                          </items>
+                        </option>
+                      </options>
+                    </option-set>
+                  </items>
+                </option>
+                <option name="option_3">
+                  <label>Option 3</label>
+                  <default>false</default>
+                  <items>
+                    <input name="imageselector" type="ImageSelector">
+                      <label>Image selector</label>
+                      <indexed>true</indexed>
+                      <occurrences minimum="1" maximum="1"/>
+                      <config>
+                        <relationship>system:reference</relationship>
+                        <allowType>mytype</allowType>
+                        <allowType>mytype2</allowType>
+                        <allowPath>path1</allowPath>
+                        <allowPath>path2</allowPath>
+                      </config>
+                    </input>
+                  </items>
+                </option>
+                <option name="option_4">
+                  <label>Option 4</label>
+                  <default>false</default>
+                  <items>
+                    <input name="double" type="Double">
+                      <label>Double</label>
+                      <indexed>true</indexed>
+                      <occurrences minimum="1" maximum="1"/>
+                    </input>
+                    <input name="long" type="Long">
+                      <label>Long</label>
+                      <indexed>true</indexed>
+                    </input>
+                  </items>
+                </option>
+              </options>
+            </option-set>
+          </form>
+        </content-type>`;
+
+exports.updateWithForm = function () {
+    let result = schemaLib.updateSchema({
+        name: 'myapp:mydata',
+        type: 'CONTENT_TYPE',
+        resource: resource
+    });
+
+    assert.assertJsonEquals({
+        "name": "myapp:mydata",
+        "displayName": "My Tag",
+        "displayNameI18nKey": "",
+        "description": "My description",
+        "descriptionI18nKey": "",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9936826)
```suggestion
        'descriptionI18nKey': "",
```